SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -285,6 +285,17 @@ class SchemaValidator
 	}
 
 	int FreeId = 10000;
+
+	// This is needed to ensure the schema generated is the same for both Windows and macOS.
+	// The new-line characters differ which will fail the tests when running it on macOS.
+	FString CleanSchema(const FString& SchemaContent)
+	{
+		FString Content = SchemaContent;
+		Content.ReplaceInline(TEXT("\r"), TEXT(""));
+		Content.ReplaceInline(TEXT("\n"), TEXT(""));
+		Content.ReplaceInline(TEXT(" "), TEXT(""));
```

Comment:
even spaces?

Reply:
probably not, let me try to remove that line